packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -401,6 +401,28 @@ class Focus extends StatefulWidget {
   /// {@endtemplate}
   final bool canRequestFocus;
 
+  /// {@template flutter.widgets.Focus.overrideFocusability}
+  /// If true, setting [canRequestFocus] on this widget to false will disable
+  /// focus for its descendants.
+  ///
+  /// Defaults to false.  Set to true if you want this widget's descendants to
+  /// be unfocusable when this widget's [canRequestFocus] is false.
+  ///
+  /// If any descendatns are focused when this is set while [canRequestFocus] is
+  /// false, they will be unfocused. When `overrideFocusability` is set to false
+  /// again, they will not be refocused, although they will be able to accept
+  /// focus again.
+  ///
+  /// See also:
+  ///
+  ///  * [ExcludeFocus], a widget that uses this property to conditionally
+  ///    exclude focus for a subtree.
+  ///  * [FocusTraversalGroup], a widget used to group together and configure
+  ///    the focus traversal policy for a widget subtree that has an
+  ///    `excludeFocus` parameter to conditionally block focus for a subtree.
+  /// {@endtemplate}
+  final bool overrideFocusability;
```

Comment:
Would it be more clear or too verbose to call this `overrideDescendantFocusability` instead?

Reply:
Yeah, I considered that, and it feels too verbose to me. I also considered `overrideCanRequestFocus` and `excludeFocus`, but they didn't really seem right either.  The name is a compromise, to be sure.